dev/workflow/rocoto/rocoto.py

```diff
@@ -429,12 +433,10 @@ def create_dependency(dep_condition=None, dep=[]) -> List[str]:
         if dep_condition is not None:
             strings.append(f'<{dep_condition}>')
 
+        # Flatten any nested lists/tuples and return plain XML strings (no leading tabs)
```

Comment:
Just a suggestion on this comment:

```suggestion
        # Flatten any nested lists/tuples and return plain XML strings
        # Note: tabbing is handled at task creation by create_innermost_task
```